service/src/main/java/bio/terra/catalog/rawls/RawlsService.java

```diff
@@ -81,6 +85,76 @@ WorkspacesApi workspacesApi(AuthenticatedUserRequest user) {
     return new WorkspacesApi(getApiClient(user));
   }
 
+  //  public List<Entity> entityList(AuthenticatedUserRequest user, String storageSourceId)  {
```

Comment:
If we don't need this, can we remove the commented out code?

Reply:
sounds good